src/features/Apiexplorer/RequestResponseRenderer/index.tsx

```diff
@@ -33,8 +33,6 @@ function RequestResponseRenderer<T extends TSocketEndpointNames>({
     try {
       request_data = JSON.parse(reqData);
     } catch (error) {
-      console.error('Could not parse the JSON data while trying to send the request: ', error);
-      console.log(error);
```

Comment:
This `console.log(error)` we don't need indeed, so you can remove that one. But, we do need the other console.errors, to help us catch errors. Lets not remove those. Please try and find the root issue on why those errors are happening and fix it. Keep the console.errors. A tip, It seems to come from this `selected_value` that returns undefined for some reason.

Reply:
I don't think the other message is also necessary since we already have a modal for it

Reply:
So I added a fix for that